packages/core/src/lib/GooglePlay.ts

```diff
@@ -70,12 +70,7 @@ export class GooglePlay {
       packageName: string,
       retainedBundles: number[],
   ): Promise<void> {
-    const edit = await this._googlePlayApi.edits.insert({packageName: packageName});
-    const editId = edit.data.id;
-    if (!editId) {
-      // TODO(@nohe427): Try to recover and understand instances where this might fail.
-      throw new Error('Could not create a Google Play edit');
-    }
+    const editId = await this.startPlayOperation(packageName);
```

Comment:
Do we not need a corresponding `endPlayOperation` call?

Reply:
I don't believe so, because these edits are committed in the end.